wrangling/bind_rows.md

```diff
@@ -6,11 +6,19 @@ Bind multiple data frames by row.
 ### From + (plus) Button
 ![](images/command-bind-rows-df-menu.png)
 
-* Click "+" button and select "Other" - > "Bind ..." -> "Rows"
+* Click "+" button and select "Merge (Add Rows / Union)"
 
 ## Bind multiple data frames by row
 
 ![](images/bind_rows.png)
 
-1. Select Data Frames that you want to bind.
-2. Column Name for Data Frame ID - Data frames identifier. When .id is supplied, a new column of identifiers is created to link each row to its original data frame. The labels are taken from the named arguments to bind_rows(). When a list of data frames is supplied, the labels are taken from the names of the list. If no names are found a numeric sequence is used instead.
+1. Select Operation Type for the Merge. Supported options are:
+
+- Bind Rows (Merge all rows)
+- Union (Merge only unique rows)
+- Intersection (Merge only the common)
+- Setdiff (Merge only the difference)
+
+2. Select Data Frames that you want to bind.
+3. Reserve Data Frame Name - When this checkbox is checked, Data Frame Name is automatically added as a new ID column.
+4. Force Data Type to Match - When this checkbox is checked, it allows you to merge data frames whose column data types are different. After the merge, it detects and assigns appropriate data types to these columns.
```

Comment:
4. (Optional) You can enable 'Force Data Type to Matc' to merge the data frames even when the data types of the corresponding columns are different. After the merge, it detects and assigns appropriate data types automatically.

Reply:
Updated as suggested.